resizing/scf-resizing/Thumbnail.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+# The user needs to create two COS Buckets, one for uploading the original pictures, and configurated as a trigger of SCF and the other for receiving the compressed pictures.
+# 用户需要创建2个COS Bucket，一个用于上传原始图片，并配置SCF为该COS Bucket触发，另一个用于接收压缩后的图片
+
+import os
+from PIL import Image
+import datetime
+from qcloud_cos_v5 import CosConfig
+from qcloud_cos_v5 import CosS3Client
+from qcloud_cos_v5 import CosServiceError
+from qcloud_cos_v5 import CosClientError
+import sys
+import logging
+
+logging.basicConfig(level=logging.INFO, stream=sys.stdout)
+
+appid = XXXXXX  # Please replace with your APPID. 请替换为您的 APPID
+secret_id = u'**************'  # Please replace with your SecretId. 请替换为您的 SecretId
```

Comment:
Maybe we can config SCF environment variables in `serverless.yml` and git it from`os.environ.get("secret_id")`.

Reply:
I think that's a great idea, will update it later